<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HDC3-D-434RPXW" name="Test" tests="2" failures="1" timestamp="28 Aug 2018 09:32:16 GMT" time="63.384" errors="0">
  <testcase name="testCase_01" time="5.82" classname="tests.Login">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at tests.Login.testCase_01(Login.java:39)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- testCase_01 -->
  <testcase name="testCase_02" time="0.0" classname="tests.Login">
    <skipped/>
  </testcase> <!-- testCase_02 -->
</testsuite> <!-- Test -->
